### PR TITLE
How To Use Fluttrer Desktop

### PR DESCRIPTION
**Pull request by shubhabanerjeewin8**
I want to help that people who want to  use flutter desktop.
Thats why i write some helpful uses way in README.md.

### CODE REVIEW

**Comment by shubhabanerjeewin8** [comment]
@googlebot I signed it

File: README.md [hunk 1]
```diff
@@ -69,3 +69,34 @@ list](https://groups.google.com/forum/#!forum/flutter-desktop-embedding-dev).
...
   in early stages, and not intended for production use.
+
+# How to run Flutter desktop application in Windows
+
+    1 Switch to Master Channel
+    2 Enable Flutter Desktop
+    3 Clone the Runner Project
+    
+
+    1 How To Switch Master Channel ?
```
Comment (stuartmorgan) [inline README.md]: This duplicates instructions linked to above. We explicitly avoid duplicating instructions, as it is a maintenance issue.

File: README.md [hunk 2]
```diff
@@ -69,3 +69,34 @@ list](https://groups.google.com/forum/#!forum/flutter-desktop-embedding-dev).
...
+    =>  Launch the Command Prompt and type  'flutter channel master' 
+    2 How To Enable Flutter Desktop ?
```
Comment (stuartmorgan) [inline README.md]: Same here.

File: README.md [hunk 3]
```diff
@@ -69,3 +69,34 @@ list](https://groups.google.com/forum/#!forum/flutter-desktop-embedding-dev).
...
+    =>  In your Command Prompt type   'flutter config --enable-windows-desktop' and 'set ENABLE_FLUTTER_DESKTOP=true'
```
Comment (stuartmorgan) [inline README.md]: The second part of this is inaccurate. That environment variable does nothing in current versions.

File: README.md [hunk 4]
```diff
@@ -69,3 +69,34 @@ list](https://groups.google.com/forum/#!forum/flutter-desktop-embedding-dev).
...
+        after that you can check that how much devices are conected .
+        Make sure that there is at least one device.
+        As Example - 'Windows * Windows * windows-x64 * Microsoft Windows [Version 10.x.xxxxx]'
+        
+    3 How To Clone The Runner Project ?
```
Comment (stuartmorgan) [inline README.md]: Teaching people how to use GitHub is beyond the scope of this project.

File: example/windows/window_configuration.cpp [hunk 5]
```diff
@@ -14,7 +14,7 @@
 
 #include "window_configuration.h"
 
-const wchar_t *kFlutterWindowTitle = L"Flutter Desktop Example";
+const wchar_t *kFlutterWindowTitle = L"Flutter Desktop Example";  //Change 'Flutter Desktop Example' to the Your Application Name
```
Comment (stuartmorgan) [inline example/windows/window_configuration.cpp]: This is already explained in a README with other information about customization.

**Pull request closed without merging**